orchagent/aclorch.h

```diff
@@ -227,6 +241,7 @@ class AclTable {
     AclTable()
         : type(ACL_TABLE_UNKNOWN)
         , m_oid(SAI_NULL_OBJECT_ID)
+        , stage(ACL_STAGE_INGRESS)
```

Comment:
>ACL_STAGE_INGRESS [](start = 16, length = 17)

default as ACL_STAGE_UNKNOWN?

Reply:
It would be better to default as ACL_STAGE_INGRESS. As the default behavior is to create ingress ACL is no stage is claimed.